ui/pluginSample/js/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "plugin-sample",
+  "version": "1.0.0",
+  "description": "test",
+  "main": "../target/scala-2.11/classes/customField.js",
+  "module": "../target/scala-2.11/classes/customField.es.js",
+  "jsnext:main": "../target/scala-2.11/classes/customField.es.js",
+  "engines": {
+    "node": ">=8",
+    "npm": ">=5"
+  },
+  "scripts": {
+    "test": "cross-env CI=1 react-scripts test --env=jsdom",
+    "test:watch": "react-scripts test --env=jsdom",
+    "build": "rollup -c",
+    "start": "rollup -c -w",
+    "prepare": "yarn run build",
+    "predeploy": "cd example && yarn install && yarn run build",
+    "deploy": "gh-pages -d example/build"
+  },
+  "peerDependencies": {
+    "react": "^15.0.0 || ^16.0.0",
```

Comment:
Can we use transitive deps to enforce using the same libs version like in main app?

Reply:
for now I added TODO